repl/scala-2.11/src/main/scala/org/apache/spark/repl/Main.scala

```diff
@@ -32,7 +34,9 @@ object Main extends Logging {
   val outputDir = Utils.createTempDir(rootDir)
   val s = new Settings()
   s.processArguments(List("-Yrepl-class-based",
-    "-Yrepl-outdir", s"${outputDir.getAbsolutePath}", "-Yrepl-sync"), true)
+    "-Yrepl-outdir", s"${outputDir.getAbsolutePath}",
+    "-Yrepl-sync",
```

Comment:
repl-sync is ineffective(i.e. it does not do anything) and deprecated we can remove it.